pkg/doc/jwt/jwt_test.go

```diff
@@ -0,0 +1,749 @@
+/*
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package jwt
+
+import (
+	"crypto/ed25519"
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/square/go-jose/v3/jwt"
```

Comment:
suggest we separate tests against the square library into their own file.

Reply:
Good idea. It becomes a sort of integration test. I will separate it.